avro-builder/builder/src/main/java/com/linkedin/avroutil1/builder/operations/codegen/own/AvroUtilCodeGenOp.java

```diff
@@ -100,10 +102,13 @@ public void run(OperationContext opContext) throws Exception {
     AvroParseContext context = new AvroParseContext();
     Set<AvscParseResult> parsedFiles = new HashSet<>();
 
-    //build a classpath SchemaSet if classpath (cp) lookup is turned on
-    ClasspathSchemaSet cpLookup = null;
-    if (config.isIncludeClasspath()) {
-      cpLookup = new ClasspathSchemaSet();
+    SchemaSet lookupSchemaSet = null;
+    if (config.getResolverPath() != null) {
+      //build a resolver path SchemaSet if ResolverPath is set
+      lookupSchemaSet = new ResolverPathSchemaSet(config.getResolverPath());
+    } else if (config.isIncludeClasspath()) {
```

Comment:
we either document that CP and path are mutually exclusive, or we support both?

Reply:
and if we choose to make them mutually exclusive - providing both arguments to the main() method should fail